src/main/java/cz/muni/fi/pa165/bluebat/entity/Customer.java

```diff
@@ -1,5 +1,95 @@
-package cz.muni.fi.pa165.bluebat.entity;
+package cz.muni.fi.pa165.bluebat.entity;/* created by rudolf */
 
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.Objects;
+
+/**
+ * @author rudolf
+ */
+@Entity
 public class Customer {
+    public Customer() {
+
+    }
+
+    public Customer(Long number) {
+
+        this.phoneNumber = number;
+    }
+
+    @OneToOne
```

Comment:
Zde záleží jak se tedy dohodneme, buď to zůstane OneToOne, nebo to bude potřeba přepsat na OneToMany (a podobně v mém PR na Reservation na ManyToOne)